### PR TITLE
[android] remove some redundant code

### PR DESCRIPTION
**Pull request by krpiotrek**

### CODE REVIEW

File: android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java [hunk 1]
```diff
@@ -300,10 +300,8 @@ private void handleFoundDevice(final ScanResult scanResult, ParcelUuid filteredU
       scanLeDevice(false);
       mUriBeaconConfig = new UriBeaconConfig(getActivity(), new UriBeaconConfigCallback(),
                                              filteredUuid);
-      if (mUriBeaconConfig != null) {
```
Comment (g-ortuno) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java]: I don't think you can do this. There are many cases in which mUriBeaconConfig could be null.

Comment (krpiotrek) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java]: It's being created one line above, and it's never nulled - am I missing something?

Comment (g-ortuno) [inline android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BeaconConfigFragment.java]: Ah you're right. Sorry, got confused with `ConfigUriBeacon.createConfigUriBeacon()`

**Comment by cco3** [comment]
Thanks!

**Pull request merged**